client/landing/jetpack-cloud/components/scan-history-item/style.scss

```diff
@@ -0,0 +1,65 @@
+.scan-history-item {
+	&.is-fixed {
+		border-left: 3px solid var( --studio-jetpack-green-40 );
```

Comment:
Instead of using the `--studio-jetpack-green-40` can we use the variables that are now available since https://github.com/Automattic/wp-calypso/pull/39878 got merged.

Reply:
Sure we can!